types/setting.go

```diff
@@ -255,6 +258,18 @@ var (
 		Default:     "300",
 	}
 
+	SettingDefinitionFailedBackupTTL = SettingDefinition{
+		DisplayName: "Failed Backup Time to Live",
+		Description: "In minutes. This setting determines how long Longhorn will keep the backup resource that was failed. Set to 0 to disable the auto-deletion.\n" +
+			"Failed backups will be checked and cleaned up during backupstore polling which is controlled by **BackupstorePollInterval** setting. " +
+			"Disabling **Backupstore Poll Interval** also means to disable failed backup auto-deletion.\n\n",
```

Comment:
Use **Backupstore Poll Interval** instead of **BackupstorePollInterval**, just for consistency.

Reply:
Modified. Thanks.